kuma/dashboards/views.py

```diff
@@ -26,7 +26,7 @@ def revisions(request):
     revisions = (Revision.objects.prefetch_related('creator__bans',
                                                    'document',
                                                    'akismet_submissions')
-                                 .order_by('-created')
+                                 .order_by('-id')
```

Comment:
It results in the same query (``ORDER BY `wiki_revision`.`id` DESC``) either way, so I'm not sure how it would be faster.